snowshu/core/docker.py

```diff
@@ -87,28 +87,29 @@ def startup(self,   # noqa pylint: disable=too-many-arguments
                 target_adapter: Type['BaseTargetAdapter'],
                 source_adapter: str,
                 envars: list,
+                base_image: str,
                 protocol: str = "tcp") -> docker.models.containers.Container:   # noqa pylint: disable=unused-argument
 
         container = self.get_stopped_container(
             image,
             start_command,
             envars,
             port,
-            name=DOCKER_TARGET_CONTAINER,
+            name=base_image,
```

Comment:
We should probably leave the container name alone for consistency and a more convenient short hand than using the full image names for the container name.